workspaces/ui-v2/src/pages/docs/components/DocPageNavigation.tsx

```diff
@@ -22,6 +23,7 @@ export const DocsPageAccessoryNavigation: FC = () => {
       {appConfig.documentation.allowDescriptionEditing && (
         <EditContributionsButton />
       )}
+      {appConfig.sharing.enabled && <ShareButton />}
```

Comment:
I think this might be better placed to on on the left, rather than on the right? i.e. the order of these buttons are - share button, changelog dropdown, edit contributions?

I don't really have a good reason other than it's my preference so feel free to disagree with me here

Reply:
I have even less of an opinion, so I shall defer to your slightly stronger opinion. changed!